enterprise/static/enterprise/admin/manage_learners.js

```diff
@@ -18,5 +55,20 @@ $(
                 })
             }
         });
+        $("#id_course").on('input', function(evt){
+            /*
+            We have to do two-step input/blur because some browsers (looking at you, Safari)
+            don't properly send the .change() event when the input is filled with autocomplete.
+            */
+            evt.target.dirty = true;
+        });
+        $("#id_course").blur(function(evt){
+            var target = evt.target;
+            if (target.dirty){
+                loadCourseModes();
+                target.dirty = false;
+            }
+        });
+        loadCourseModes()
```

Comment:
what is the purpose of double calling `loadCourseModes()` ? 
nit: loadCourseModes()`;`